File: 프로그래머스/unrated/181912. 배열 만들기 5/배열 만들기 5.java
class Solution {
    public int[] solution(String[] intStrs, int k, int s, int l) {
        int len = intStrs.length;
        int[] tmp = new int[len];
        int count = 0;
        for (int i = 0; i < len; i++)
        {
            int num = Integer.parseInt(intStrs[i].substring(s, s + l));
            if (k < num)
                tmp[count++] = num;
        }
        int[] answer = new int [count];
        System.arraycopy(tmp, 0, answer, 0, count);
        return answer;
    }
}

File: 프로그래머스/Lv.0/181912. 배열 만들기 5/배열 만들기 5.js
function solution(intStrs, k, s, l) {
    var answer = [];
    for (var i = 0; i < intStrs.length; i++) {
        var num = Number(intStrs[i].substring(s, s + l));
        if (k < num)
            answer.push(num);
    }
    return answer;
}